ui/desktop/app/templates/scopes/scope/projects/targets/target/hosts.hbs

```diff
@@ -5,35 +5,37 @@
   <Rose::Table as |table|>
     <table.body as |body|>
       {{#each @model.hosts as |host|}}
-        <body.row as |row|>
-          <row.cell>
-            {{host.displayName}}
-            <p>{{host.description}}</p>
-          </row.cell>
-          <row.cell>
-            <Copyable
-              @text={{host.id}}
-              @buttonText={{t 'actions.copy-to-clipboard'}}
-              @acknowledgeText={{t 'states.copied'}}
-            >
-              <code>{{host.id}}</code>
-            </Copyable>
-          </row.cell>
-          <row.cell>{{host.attributes.address}}</row.cell>
-          <row.cell>
-            <Rose::Badge>{{t
-                (concat 'resources.host.types.' host.type)
-              }}</Rose::Badge>
-          </row.cell>
-          <row.cell>
-            <Rose::Button
-              @style='secondary'
-              {{on 'click' (route-action 'connect' @model.target host)}}
-            >
-              {{t 'resources.session.actions.connect'}}
-            </Rose::Button>
-          </row.cell>
-        </body.row>
+        {{#if host.isStatic}}
```

Comment:
In Desktop we **do** want to support dynamic hosts.  Let's make sure they are visible, drop the "type" column, and double check that we can display address/addresses correctly.

Reply:
Ok, this is my bad I fully remember now we said to drop the type column.